gadgets/cmr/config/LandmarkDetection/gadgetron_cmr_landmark_detection_util.py

```diff
@@ -162,3 +235,17 @@ def adaptive_thresh_cpu(probs, p_thresh=0.5, p_thresh_max=0.988):
         mask = np.zeros((RO,E1))
 
     return mask
+
+if __name__=="__main__":
```

Comment:
As I understand it, you don't actually run this as a main function, so maybe we should not have this section? And if you do want it, I don't think baking the hash of a specific model in here is the best, you are not using those in validation from the C++ code, you use the Gadget parameters there. I would make them parameters here or remove this section.